atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -72,6 +77,22 @@ def get_patched_release(self, original_release, increment=False):
         return '.'.join([part for part in [release, suffix, rest]
                          if part is not None])
 
+    def get_next_release_general(self, component, version, release, release_label,
```

Comment:
Why `get` method is not returning release value, but hides it directly in release label. This side effects seems hard to read to me.

Reply:
renamed